<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="IN2253656W1" failures="1" tests="6" name="VerifyCounterFunctionality" time="57.27" errors="0" timestamp="2021-03-05T19:05:45 IST">
  <testcase classname="com.testClasses.TC_002_VerifyCounterWorks" name="counterTest" time="1.443"/>
  <testcase classname="com.testClasses.TC_003_VerifyCountIsCorrect_DataDriven" name="Test" time="1.428"/>
  <testcase classname="com.testClasses.TC_003_VerifyCountIsCorrect_DataDriven" name="Test" time="1.292"/>
  <testcase classname="com.testClasses.TC_003_VerifyCountIsCorrect_DataDriven" name="Test" time="1.195"/>
  <testcase classname="com.testClasses.TC_003_VerifyCountIsCorrect_DataDriven" name="Test" time="1.352">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testClasses.TC_003_VerifyCountIsCorrect_DataDriven.Test(TC_003_VerifyCountIsCorrect_DataDriven.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test -->
  <testcase classname="com.testClasses.TC_003_VerifyCountIsCorrect_DataDriven" name="Test" time="1.65"/>
  <testcase classname="com.testClasses.BaseClass" name="@BeforeSuite suitStartSetup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite suitStartSetup -->
</testsuite> <!-- VerifyCounterFunctionality -->
